### PR TITLE
added edit mode hints

### PR DESCRIPTION
**Pull request by logan-anderson**

### CODE REVIEW

**Review by spbyrne — DISMISSED** [review]
👍

File: content/guides/tina-cloud/starter/connect-to-tina-cloud.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ This time a modal asks you to authenticate through TinaCMS. Upon success, Tina w
 Make some edits through the sidebar and click save.
 Changes are saved in your GitHub repository.
 
+> Hint: To exit edit mode navigate to the `/exit-admin` route if your running on localhost it is http://localhost:3000/exit-admin .
```
Comment (Enigmatical) [inline content/guides/tina-cloud/starter/connect-to-tina-cloud.md:37]:
```suggestion
> Hint: To exit edit mode, navigate to the `/exit-admin` route.  If you are running on `localhost`, it is `http://localhost:3000/exit-admin`.
```

File: content/guides/tina-cloud/starter/local.md [hunk 2]
```diff
@@ -42,3 +42,6 @@ Next, click on the pencil in the lower left-hand corner to open Tina's sidebar w
 ![tina-cloud-starter](/img/tina-cloud-starter.jpg)
 
 Make some changes, build your own components and experience contextual editing with Tina. As fun as this is, you'll likely want to collaborate with others and view changes on a publicly available URL. To do that, in the next step we'll connect to Tina Cloud.
+
+ 
+> Hint: To exit edit mode navigate to the `/exit-admin` route if your running on localhost it is http://localhost:3000/exit-admin .
```
Comment (Enigmatical) [inline content/guides/tina-cloud/starter/local.md:47]:
```suggestion
> Hint: To exit edit mode, navigate to the `/exit-admin` route.  If you are running on `localhost`, it is `http://localhost:3000/exit-admin`.
```

**Review by Enigmatical — DISMISSED** [review]
Cleaning up the grammar a bit, but otherwise, this looks good.

**Review by jamespohalloran — APPROVED** [review]

**Review by DirtyF — APPROVED** [review]

**Pull request merged**